src/components/home/Features.tsx

```diff
@@ -0,0 +1,52 @@
+import { features } from '@site/src/constants';
+import GtU from '@site/static/icons/basic/gt-underscore.svg';
```

Comment:
Can you change the name from GtU to a more readble one like 
GreaterThanUnderscoreIcon
whatever icons we import let's include the word "Icon" in them